test/test_example_parsers/test_dice_roll_parser.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from plusminus.examples.example_parsers import DiceRollParser
+
+
+@pytest.fixture
+def die():
+    return DiceRollParser()
```

Comment:
This could be in a conftest.py, but since you said you might spin one or more of these off into their own repo, I've kept it with the tests for the parser. Pytest will search in the test script first, then elsewhere.

Reply:
I think you'd keep conftest.py reserved for those test features that are not parser-specific. Not sure I understand the value of this fixture, but it is probably a pytest idiom.

Reply:
In this case, it's not doing much work except saving a couple of lines of setup, and if there's an error building the parser, then the error is highlighted as an error (in the fixture) rather than a test fail.
eg if you try:
```python3
@pytest.fixture
def die():
    import random
    if random.randint(0, 10)  % 2 == 0:
        raise ValueError
    return DiceRollParser()
```
and then run `python -m pytest -vv` for verbose output, you see all the PASS/FAIL/ERROR codes, except you won't see any failing tests. 

Writing it this way also lets you change the call if the name or something about `DiceRollParser` changes, say you need to pass an argument to it in the 'new version', you can just change it in the fixture rather than everywhere in the code. 

I believe you could scope it to be class or module scoped, such that it would cache the parser object rather than passing a new object for each case, which is what I believe it is doing now, which will get you some speedup, or would be useful if you wanted to use a fresh database for a series of tests.